Model/SongMeta.cs

```diff
@@ -18,6 +18,7 @@ public class SongMeta
     public Guid ContentGuid;
     public string GuidString => ContentGuid.ToString().ToUpper();
     public string ZipFileName => Path.Combine(Properties.Settings.Default.TrackPath, GuidString + ".zip");
+    public string DirName => Path.Combine(Properties.Settings.Default.TrackPath, GuidString + ".jcf");
```

Comment:
This won't work between windows and mac. `ZipFileName` and `DirName` should be combined into one field, `SongPath` or something, which holds the full path to the .zip/folder. Then a flag can indicate whether the Song is a Zip song or a Folder song. These should then be added to the XML caching functions below